contracts/MetadataViews.cdc

```diff
@@ -11,113 +12,38 @@ import NonFungibleToken from "NonFungibleToken"
 /// a different kind of metadata, such as a creator biography
 /// or a JPEG image file.
 ///
-pub contract MetadataViews {
-
-    /// Provides access to a set of metadata views. A struct or
-    /// resource (e.g. an NFT) can implement this interface to provide access to
-    /// the views that it supports.
-    ///
-    pub resource interface Resolver {
```

Comment:
Do we need to add back `access(all) resource interface Resolver {}` and `access(all) resource interface ResolverCollection {}` for backwards compatibility with existing resources that conform to them?

Reply:
AFAIK, those got removed here because they moved to `ViewResolver`

Reply:
yep, and we already have the upgrade exception planned for the migration